views/components/Graphs/BarGraph.jsx

```diff
@@ -24,18 +24,21 @@ const tooltipText = (graph, datum) => {
   const { study, studyTarget, count, valueLabel } = datum
   if (graph.studyTotals[study]) {
     const { targetTotal, count: studyTotalCount } = graph.studyTotals[study]
+    const totalCount = targetTotal || studyTotalCount
     const showToolTip =
       study && count && study !== TOTALS_STUDY && valueLabel !== NOT_AVAILABLE
     return showToolTip
-      ? `${study} target: ${targetTotal} (100%)\n${study} current: ${studyTotalCount} (${toolTipPercent(
+      ? `${study} target: ${targetTotal || 'N/A'} (100%)\n${study} current: ${
+          studyTotalCount || 'N/A'
+        } (${toolTipPercent(
           studyTotalCount,
-          targetTotal
-        )}%)\n${valueLabel} target: ${studyTarget} (${toolTipPercent(
+          totalCount
+        )}%)\n${valueLabel} target: ${studyTarget || 'N/A'} (${toolTipPercent(
           studyTarget,
-          targetTotal
-        )}%)\n${valueLabel} current: ${count} (${toolTipPercent(
+          totalCount
+        )}%)\n${valueLabel} current: ${count || 'N/A'} (${toolTipPercent(
```

Comment:
🎱 should `'N/A'` be a constant?

Reply:
it sure should, it even already is! I'll use it here, thanks.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/159/commits/b63f19967007ff3d1e29212aae48f1e0d21ca548